prime-router/metadata/schemas/covid-19.schema

```diff
@@ -405,35 +405,38 @@ elements:
 
   - name: patient_age
     type: NUMBER
+    cardinality: ZERO_OR_ONE
     natFlatFileField: Patient_age
     hhsGuidanceField:
     hl7Field: AOE
     hl7AOEQuestion: 30525-0
 
   - name: patient_age_units
     type: CODE
+    cardinality: ZERO_OR_ONE
     valueSet: patient_age_units
     natFlatFileField: Patient_age_units
     hhsGuidanceField:
     documentation: Always filled when `patient_age` is filled
 
   - name: patient_city
     type: CITY
-    cardinality: ONE
+    cardinality: ZERO_OR_ONE
     natFlatFileField: Patient_city
```

Comment:
Changed or added cardinality to many patient_ fields, because we've seen in real data that they are sometimes blank.    This definitely needs more work.

I only added this for patient_ related fields.